Automattic/simplenote-android
# Update Selected Color

### Fix
Update the selected background color of the ***Light*** theme from `blue_0` (`#eff3f7`) to `blue_5` (`#c6def6`) for higher contrast and better accessibility.  See the screenshots below for illustration.

![update_selected_color](https://user-images.githubusercontent.com/3827611/64368526-c023ca00-cfd7-11e9-8656-238d9ebd9ff8.png)

### Test
0. Set ***Theme*** to ***Light***.
1. Open navigation drawer.
2. Notice selected background color is `blue_5` (`#c6def6`).
3. Close navigation drawer.
4. Long-press any note in list.
5. Notice selected background color is `blue_5` (`#c6def6`).
6. Notice app bar background color is `blue_5` (`#c6def6`).

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
